gradle/testing/defaults-tests.gradle

```
@@ -181,6 +181,13 @@ allprojects {
         // Print some diagnostics about locations used.
         logger.info("Test folders for {}: cwd={}, tmp={}", project.path, testsCwd, testsTmpDir)
       }
+      if(isCIBuild) {
```

We have yet to apply the Google Java Style format to build files (we should do this for Solr 10!) but please try to adhere to it.
```suggestion
      if (isCIBuild) {
```